### DISCUSSION TITLE
Add option to autostart applications in DropDown/scratchpad

### DISCUSSION THREAD

**Discussion by hanschen** [qtile/qtile#4106] [Ideas]
It would be nice if there was an option to automatically run the command specified in `DropDown`, so that one can autostart applications in scratchpads. Currently I simulate the scratchpad key binding in a `@hook.subscribe.startup_complete` hook, which seems sub-optimal.

See also this discussion: https://www.reddit.com/r/qtile/comments/v4003j/scratchpad_autostart/

**Comment by m-col** [comment]
Yeah that would be nice. Something like the `init` option of the `Group` class in config.py, which flags whether the group should be created at startup. The same thing for `DropDown` to indicate that the dropdown process should be spawned at startup (but still be hidden when doing so) could be useful. PR welcome :)
